cpp/src/io/shp/polygon_shapefile_reader.cpp

```diff
@@ -32,8 +32,7 @@ cudf::size_type read_ring(OGRLinearRing const& ring,
 {
   cudf::size_type num_vertices = ring.getNumPoints();
 
-  // append points in reverse order
-  for (cudf::size_type i = num_vertices - 1; i >= 0; i--) {
+  for (cudf::size_type i = 0; i < num_vertices; ++i) {
```

Comment:
I interpreted @zhangjianting 's comment here: https://github.com/rapidsai/cuspatial/pull/609#issuecomment-1204575080 to mean that clockwise vs. counterclockwise is a matter of choice since the OGC and ESRI shapefile formats are opposites. In which case why not either leave this as-is, or provide a parameter to specify which is expected?

Reply:
Providing a parameter might be the way to go here, just in case. In this context, GeoPandas is used to read a shapefile and `read_polygon_shapefile` is used to read a shapefile. While both are valid technically, the coordinates are wound in the opposite direction for the result that is produced by `read_polygon_shapefile` which is why I want to introduce this change.